### DISCUSSION TITLE
Lazy loading 'index.js'

### DISCUSSION THREAD

**Discussion by LaundroMat** [quasarframework/quasar#16087] [General - Components / Directives / etc]
I'm already lazy loading all components in `routes.js`. I'm not lazy loading any components in my source files, however.

When building my app, I get a warning that " Some chunks are larger than 500 KiB after minification." 
The chunk in question is called `assets/index.4178f8b8.js`.  What file is that? Where should I look in my code to lazy load components?

**Comment by freddy38510** [comment]
The index.js file is the chunk corresponding to the bundle's entry point, which is originally the index.html file [transformed by Vite](https://github.com/vitejs/vite/blob/v2.9.16/packages/vite/src/node/plugins/html.ts#L221). You can configure the [chunking strategy](https://v2.vitejs.dev/guide/build.html#chunking-strategy).

But first, it would be a good idea to analyze the production bundles.

```js
// quasar.config.js

const { configure } = require('quasar/wrappers');

module.exports = configure(function (/* ctx */) {
  return {
    // Full list of options: https://v2.quasar.dev/quasar-cli-vite/quasar-config-js#build
    build: {
      // ...
      /**
       * Do you want to analyze the production bundles?
       * Generates and opens an html report.
       * @default false
       */
      analyze: true
    },
  }
});
```
